src/en/commands.md

```diff
@@ -1167,10 +1167,8 @@ Click on the expander to see details for each command.
 
              # How long to wait for a connection to the controller
              bootstrap-timeout: 600 # default: 10 minutes
-             # How long to wait between connection attempts to a controller 
-   
```

Comment:
There were extra spaces in here that broke up the code block. I removed them. That is all.